src/containers/App/App.js

```diff
@@ -113,9 +113,9 @@ const App = (props) => {
               />
               <Route exact path="/paystrings/:id?" component={paystrings} />
               <Route exact path="/token/:currency.:id" component={token} />
-              {MODE === 'mainnet' && (
+              {/* {MODE === 'mainnet' && (
                 <Route exact path="/tokens" component={tokens} />
-              )}
+              )} */}
```

Comment:
we can just delete this. the history is in the repo and if this is reimplemented it will probably be a complete rewrite anyway. tbh we could delete the components etc as well while we're at it, but if time is a concern that's just icing :)

Reply:
yeah I can delete the components

Reply:
deleted the components